deps/gen_functions.c

```diff
@@ -57,6 +58,7 @@ int main(int argc, char *argv[])
   printf("const MPI_WAITALL            = Libdl.dlsym(libmpi, \"%s\")\n", STRING(MPI_WAITALL));
   printf("const MPI_WAITANY            = Libdl.dlsym(libmpi, \"%s\")\n", STRING(MPI_WAITANY));
   printf("const MPI_WTIME              = Libdl.dlsym(libmpi, \"%s\")\n", STRING(MPI_WTIME));
+  printf("const MPI_IN_PLACE           = Base.box(Ptr{Int}, Base.unbox(Int, Int(%d)))\n", MPI_IN_PLACE);
```

Comment:
Yea this one is weird, you can test it yourself, but the fortran MPI interface and C interface need different values for MPI_IN_PLACE, that's why I generated it using the C.  See https://software.intel.com/en-us/forums/intel-clusters-and-hpc-technology/topic/280060 as an example